### PR TITLE
Clarifications about getElementsByTagNameNS

### PR DESCRIPTION
**Pull request by nielsdos**
And while we're here, make the documentation consistent.

Never contributed to the docs before, so not sure about wording etc :P

### CODE REVIEW

File: reference/dom/domdocument/getelementsbytagnamens.xml [hunk 1]
```diff
@@ -27,8 +27,9 @@
      <term><parameter>namespace</parameter></term>
      <listitem>
       <para>
-       The namespace URI of the elements to match on. 
+       The namespace URI of the elements to match on.
        The special value <literal>*</literal> matches all namespaces.
+       Passing <type>null</type> matches the empty namespace.
```
Comment (Girgias) [inline reference/dom/domdocument/getelementsbytagnamens.xml:32]:
```suggestion
       Passing &null; matches the empty namespace.
```
For the value `null` we use the entity (we have some similarly for true and false, you can have a look in `language-snippets.ent` and another `.ent` file the ones we define)

File: reference/dom/domdocument/getelementsbytagnamens.xml [hunk 2]
```diff
@@ -27,8 +27,9 @@
      <term><parameter>namespace</parameter></term>
      <listitem>
       <para>
-       The namespace URI of the elements to match on. 
+       The namespace URI of the elements to match on.
        The special value <literal>*</literal> matches all namespaces.
```
Comment (Girgias) [inline reference/dom/domdocument/getelementsbytagnamens.xml:31]:
```suggestion
       The special value <literal>"*"</literal> matches all namespaces.
```
I would enclose it in quotes to show that we mean the string with an asteriks

**Review by Girgias — CHANGES_REQUESTED** [review]
Ditto the changes onto the other page.

**Comment by nielsdos** [comment]
Thanks, applied the suggestions.

**Review by Girgias — APPROVED** [review]
I wonder if it makes sense to deprecate the behaviour of passing ``null`` to these functions as it is rather confusing.

**Comment by Girgias** [comment]
So the spec is just weird :| yeah let's follow the spec.

**Pull request merged**